spec/helpers/sessions.rb

```diff
@@ -0,0 +1,21 @@
+module SessionHelpers
+
+  def sign_up (user)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis.